framework-cloud/framework-cloud-common/src/main/java/org/kie/cloud/common/logs/InstanceLogUtil.java

```diff
@@ -61,4 +62,17 @@ public static void writeDeploymentLogs(DeploymentScenario<?> deploymentScenario)
             }
         }
     }
+
+    private static File getOutputFile(String instanceName, String customLogFolderName) {
+        File outputDirectory = new File(System.getProperty(INSTANCES_LOGS_OUTPUT_DIRECTORY, DEFAULT_LOG_OUTPUT_DIRECTORY));
+        if (!outputDirectory.isDirectory()) {
+            outputDirectory.mkdir();
+        }
+        outputDirectory = new File(outputDirectory, customLogFolderName);
```

Comment:
Let's not reuse the same variable for a different directory, that's misleading for a reader.

If you need to make sure the directory structure leading to your directory has been created, create the target directory and call `mkdirs()` instead of `mkdir()` - notice the plural form.

Reply:
I just extracted some code from previous.
But modified it with new variable name.

I still need to modify the mkdirs

Reply:
updated